client/src/redux/actions/tests/scansActions.test.js

```diff
@@ -0,0 +1,82 @@
+import configureMockStore from 'redux-mock-store';
+import promiseMiddleware from 'redux-promise-middleware';
+import moxios from 'moxios';
+import expect from 'expect';
+
+import { scansTypes } from '../../constants';
+import * as actions from '../scansActions';
+
+const middlewares = [promiseMiddleware()];
+const mockStore = configureMockStore(middlewares);
+
+describe('ScansActions', function() {
+  beforeEach(() => {
+    process.env.REACT_APP_ENV = 'development';
```

Comment:
Note: Lets iterate on this... think you can avoid having to do this, take a look at the NPM script

```
"start:ui-stage": "sh -ac '. .env.staging; react-app-rewired start'",
```